### ISSUE TITLE
[1.15.2] CIT packs currently not working in pre-release version

### ISSUE THREAD

**Issue by aliensonhigh** [sp614x/optifine#3276]
![2020-02-13_23 08 14](https://user-images.githubusercontent.com/61023620/74462630-9049ee00-4eb6-11ea-8b8d-aa4c59558e5e.png)
This is how the CIT is supposed to look like. Im using mizuno's CIT pack on 1.14.4 .
![2020-02-13_22 48 46](https://user-images.githubusercontent.com/61023620/74462718-b53e6100-4eb6-11ea-83d8-d90e44f8ded7.png)
However when i am trying to use it in 1.15 the model shows up in the hotbar with the missing texture checkers, however uses the default texture and model when I hold it in my hand/place it on an item frame.

**Comment by TBlazeWarriorT** [comment]
Can confirm. My own resource pack doesn't work at all, I checked everything and all settings. Current 1.15.2 preview 1 doesn't support CIT at all, as it seems.

**Comment by Th3GreenGamer** [comment]
Can confirm on 1.15.2 preview 2. Same exact thing with a CIT model of a named axe and sword

**Comment by AlexandreRkh** [comment]
Can confirm on 1.15.2. Same problem with CIT in item frame and on ground renders.

**Comment by Th3GreenGamer** [comment]
Can confirm on pre3 and pre4

**Comment by Th3GreenGamer** [comment]
And pre5.

**Comment by Th3GreenGamer** [comment]
pre6 too.

**Comment by Th3GreenGamer** [comment]
Pre7 also.

**Comment by TBlazeWarriorT** [comment]
OptiFine 1.15.2_HD_U_G1_pre9: Name-based cit confirmed still broken

**Comment by Rixic** [comment]
Same here, I also noticed armor CITs don't appear to be working either (the armor itself)

**Comment by WizardVS** [comment]
same here, nothing in cit works pre9 here

**Comment by cookieXL** [comment]
Still not working in pre10

**Comment by Th3GreenGamer** [comment]
Not in pre12 either.

**Comment by tfeak2** [comment]
pre13 items working but armor isn't.

**Comment by jana1ib** [comment]
I'm still getting the missing texture checkers in pre13?

**Comment by CitrineFool** [comment]
I can't get my item texture to work in pre13. I am trying to get a tropical_fish to change to the Stardew Valley Anchovy when renamed Anchovy.

**Comment by CitrineFool** [comment]
I got it to work. The subfolder that it was in was capitalized.

**Comment by aliensonhigh** [comment]
Items are still not working for me
changed version in pack.mcmeta to '5' as well but no results. The texture wont show on the item held in hand either.
![2020-04-27_19 35 30](https://user-images.githubusercontent.com/61023620/80382075-76b62100-88bf-11ea-956d-7e36b06e7185.png)
![2020-04-27_19 41 58](https://user-images.githubusercontent.com/61023620/80382084-79187b00-88bf-11ea-93f4-32d5f94ef027.png)

**Comment by tychoflorent** [comment]
pre13 items equipped/third person and item frame not working can confirm

GUI is working however.

**Comment by Th3GreenGamer** [comment]
![2020-05-03_07 08 03](https://user-images.githubusercontent.com/49415644/80910875-1d118480-8d0d-11ea-9b75-8cc79b752eae.png)

Using a custom resourcepack with John Smith Legacy as a base. Sword should look like a 3d katana when named Occam's Razor. The model shows in GUI without a texture, but when held it defaults to the texture packs regular diamond sword texture. This is on pre14.

**Comment by WizardVS** [comment]
when are they gonna fix this, this is super annoying

**Comment by jana1ib** [comment]
Extraneous feature as it is, the best we can do is be supportive and patient, and provide constructive input to help the process along.

**Comment by aliensonhigh** [comment]
Update for pre15: same results as i had posted for pre13, model appears in gui without texture, but does not work in hand or on item frames etc.

**Comment by aliensonhigh** [comment]
pre16 update: not working

**Comment by Thr0in** [comment]
pre18 update: GUI and dropped on the floor is working (with model and texture) the other ones aren't

tl;dr same as before

**Comment by aliensonhigh** [comment]
pre19 update: same as before

**Comment by Zaddish** [comment]
Custom items, nbt data -> WORKING
Custom models -> BROKEN

This is very annoying, please fix @sp614x :/

**Comment by sp614x** [comment]
Fixed, coming in pre20.

**Comment by aliensonhigh** [comment]
![2020-06-06_15 56 42](https://user-images.githubusercontent.com/61023620/83950623-d2be7e80-a849-11ea-9d16-69eef724d563.png)
pre 21: models are working however texture doesnt seem to be appearing, Dont know if this is just an issue with the pack or optifine.

**Comment by jana1ib** [comment]
If I’m correct in assuming what’s pictured is Mizuno’s CIT, I’m pretty sure it’s the pack since its latest version is 1.14

**Comment by FrankSchepers24** [comment]
In optifine 1.15 the textures for the models need to be saved in a different location than before. I don't know exactly where, but we did fix it some time ago. So just look it up on google or something like that, it should be out there.

**Comment by jana1ib** [comment]
Got Mizuno's 1.14 CIT to work. In the resource pack, go to assets>minecraft>optifine>cit. In the .json files, the syntax `"./"` DOES NOT work. So instead of `"parent": "./cit_item",` replace with `"parent": "optifine/cit/cit_item",`  The correct textures should appear
